eos-updater/com.endlessm.Updater.xml

```diff
@@ -293,8 +293,11 @@
       @expected: Number of bytes expected.
 
       Unused.
+
+      Deprecated: Use `org.freedesktop.DBus.Properties.PropertiesChanged`.
```

Comment:
If it's not emitted by the daemon, or consumed by any existing clients, can we just drop it here?

Reply:
Probably. It’s not used by gnome-software; not sure where else it could be used.

Reply:
Dropped.